bin/deploy

```diff
@@ -255,6 +248,16 @@ main() {
 
     conda "${ARGS[@]}"
 
+    # The conda Intel MKL linked packages are intentionally avoided [on linux].
+    # See: https://jira.lsstcorp.org/browse/DM-5105 and
+    #      https://jira.lsstcorp.org/browse/DM-12266
+    if [[ $BLEED_DEPLOY == true ]]; then
+      if [[ $(uname -s) == Linux* ]]; then
+        conda install --yes nomkl
```

Comment:
If the intent is to install the `nomkl` package before anything else is installed, the ordering is wrong -- this probably should be at ~ line 235.